src/coldcard.ts

```diff
@@ -550,6 +529,18 @@ export class ColdcardSignMultisigTransaction extends ColdcardInteraction {
  *
  */
 export class ColdcardMultisigWalletConfig {
+  jsonConfig: any;
+
+  name: string;
+
+  requiredSigners: number;
+
+  totalSigners: number;
+
+  addressType: string;
```

Comment:
This is also available from `unchained-bitcoin`

```suggestion
  addressType: MultisigAddressType;
```

Reply:
Changed in 35668a8805582ecc715ddc5e5214ac4de91c22ff